src/index.js

```diff
@@ -1,6 +1,6 @@
 // Copyright 2018 Kensho Technologies, LLC.
 
-export {default as DEFAULT_THEME} from './defaultTheme'
+export {THEME as DEFAULT_THEME} from './defaults'
```

Comment:
I wonder if we need to export this, since we always merge custom theme objects with the default one. We don't consume the export internally.

Reply:
I think it's better to not export it. The README documents the `DEFAULT_THEME` as the base for the configurable theme of a chart, not necessarily as something that needs to be consumed/modified directly.

That's something that can be taken care of in a separate PR though